torch/_inductor/codegen/common.py

```diff
@@ -1723,7 +1730,7 @@ def generate(
         cache_key = expr.getvalue() if isinstance(expr, IndentedBuffer) else expr
         var = self.cache.get(cache_key, None)
         if not var:
-            var = self.newvar(bounds)
+            var = self.newvar(dtype, bounds)
```

Comment:
Just commenting on this now but.. It's better practice to add a new arg at the end of arguments, typically...